dashboard/app/views/shared/_school_info_registration.html.haml

```diff
@@ -0,0 +1,101 @@
+- require 'country_codes'
```

Comment:
This file is a variation on _school_info.html.haml - the layout is different when it appears on the registration page than it is in the other places where it's used, so unless/until we update the other layouts, duplication is the cleanest way to handle both layouts.